src/App.tsx

```diff
@@ -30,31 +96,21 @@ export const App = () => {
       </form>
 
       <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="2" className="TodoInfo">
-          <h2 className="TodoInfo__title">
-            quis ut nam facilis et officia qui
-          </h2>
-
-          <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-            Patricia Lebsack
-          </a>
-        </article>
+        {todos.map(todo => (
+          <article
+            key={todo.id}
+            data-id={todo.id}
+            className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
+          >
+            <h2 className="TodoInfo__title">{todo.title}</h2>
+            <a
+              className="UserInfo"
+              href={`mailto:${usersFromServer.find(user => user.id === todo.userId)?.email}`}
+            >
+              {usersFromServer.find(user => user.id === todo.userId)?.name}
+            </a>
+          </article>
+        ))}
```

Comment:
you need to reuse this component here
<img width="560" alt="image" src="https://github.com/user-attachments/assets/3757bcbc-aa4a-4dc8-9bc9-c44c92174b6f">